apps/saleor-app-checkout/backend/payments/updateTransaction.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from "@/saleor-app-checkout/graphql";
 
 export const updateTransaction = async (args: TransactionUpdateMutationVariables) => {
-  const { data, error } = await getClient()
+  const { data, error: _error } = await getClient()
```

Comment:
`_error` is unused, I think we can remove this

btw. does the new ESLint config ignore `no-unsed-vars` rule when the var starts with `_`?

Reply:
Yes